.github/actions/deploy/action.yaml

```
@@ -83,6 +83,11 @@ runs:
       shell: bash
       run: sh tests/smoke/deploy-sorbet-resources.sh end2end
       working-directory: ./.github/scripts/end2end/operator
+    - name: Deploy metadata
+      shell: bash
+      run: bash deploy-metadata.sh
```

We are already running with bash, so make it executable and execute just:

```suggestion
      run: ./deploy-metadata.sh
```